### ISSUE TITLE
Steam Overlay not working in El Capitan beta

### ISSUE THREAD

**Issue by bxio** [ValveSoftware/Dota-2#1816]
I've disabled and re-enabled accessibility option, and it still doesn't work. 

Edit: Doesn't work on CS:GO either.

**Comment by SaifurRahmanMohsin** [comment]
+1

**Comment by gfnord** [comment]
+1, not working

**Comment by theskillwithin** [comment]
doesn't work in cs:go

**Comment by scottlow** [comment]
I heard on another thread (which I can't find now) that this is an Apple issue as Apple changed the way accessibility was done in El Cap. Hoping a fix comes from them soon!

**Comment by SaifurRahmanMohsin** [comment]
That’s probably correct. I’m currently on El Capitan beta 5 and still it has not been fixed. Hope it gets done soon. And I also hope steam re-writes dota to run over Metal 😄 as that would be great!
